setup.py

```diff
@@ -122,6 +122,17 @@
     author_email='rbean@redhat.com',
     url='https://github.com/fedora-infra/fedmsg/',
     license='LGPLv2+',
+    classifiers=[  # https://pypi.python.org/pypi?%3Aaction=list_classifiers
+        'Development Status :: 3 - Alpha',
```

Comment:
If people are more comfortable with the X release being bumped than they are with the 0.y release being bumped, that's fine with me.

However, I'd still like the next release to be 0.19 so I have a chance to get the documentation in order and determine what API I'm declaring stable (e.g. is the on-the-wire format part of the API? What Python modules, exactly?).